main.tf

```diff
@@ -116,6 +118,73 @@ data "aws_iam_policy_document" "resource_readonly_access" {
   }
 }
 
+data "aws_iam_policy_document" "resource_readonly_lambda_access" {
```

Comment:
instead of creating a new policy document, you can use a dynamic to extend the above document